pymc/distributions/continuous.py

```diff
@@ -382,7 +382,7 @@ def logp(value):
 
         Parameters
         ----------
-        value: numeric
+        value : numeric or ndarray or aesara.tensor
```

Comment:
```suggestion
        value : tensor_like
```

Today we decided to use this alias to include array_like, scalars and tensorvariable, otherwise it would look like `numeric, array_like or TensorVariable` which gets a bit too long. same for the logcdf below.

Reply:
@OriolAbril Updated the alias.